tests/platform_tests/cli/test_show_platform.py

```diff
@@ -395,10 +408,29 @@ def test_show_platform_ssdhealth(duthosts, enum_supervisor_dut_hostname):
     pytest_assert(len(unexpected_fields) == 0, "Unexpected fields in output: {} on '{}'".
                   format(repr(unexpected_fields), duthost.hostname))
 
-    # TODO: Test values against platform-specific expected data instead of testing for missing values
     for key in expected_fields:
         pytest_assert(ssdhealth_dict[key], "Missing value for '{}' on '{}'".format(key, duthost.hostname))
 
+        line_data = ssdhealth_dict[key]
+        # Some platforms may have "N/A" value which is expected
+        is_line_empty = True if (not line_data or line_data == "N/A") else False
+        is_not_supported = True if key in unsupported_ssd_values_per_platform.get(platform, []) else False
+
+        if is_line_empty and is_not_supported:
+            logging.info("Validation ignored for '{}' on platform: '{}'".format(key, platform))
+            continue
+
+        pytest_assert(not is_line_empty, "Invalid data '{}' for '{}'".format(line_data, key))
+
+        if key == "Health":
+            health_float_value = float(line_data.strip("%"))
+            pytest_assert(health_float_value > 50.0, "SSD health is '{}', SSD replacement required".format(line_data))
```

Comment:
Suggest defining a constant to represent the meaning of 50%

Reply:
Fixed, please approve